7.figures/nbconverted/Figure3_model_evaluation.r

```diff
@@ -34,18 +38,31 @@ cm_df$Predicted_Label <-
 
 # Order feature types for plotting
 cm_df$feature_type <-
-    dplyr::recode_factor(cm_df$feature_type, !!!facet_labels)
+    dplyr::recode_factor(cm_df$feature_type, !!!feature_spaces)
 
+print(dim(cm_df))
 head(cm_df)
 
+focus_feature_spaces <- c(
+    "CellProfiler",
+    "DeepProfiler",
+    "CP and DP"
+)
+
+subset_feature_spaces <- c(
+    "CP Zernike",
+    "CP AreaShape"
+)
+
 custom_labeller <- function(value) {
   paste("Shuffled:", value)
 }
 
 confusion_matrix_gg <- (
     ggplot(
         cm_df %>%
-            dplyr::filter(data_split == "test"),
+            dplyr::filter(data_split == "test") %>%
+            dplyr::filter(feature_type %in% !!focus_feature_spaces),
```

Comment:
I think this relates to the above question! FWIW, I feel like after using Python so much, there should be a simpler way to do this without specifying a new variable, but I have lots to learn.

Reply:
dplyr is the simplest way to wrangle a dataframe in R - my updates to `focus_feature_spaces` will make this a bit more clear

Reply:
fixed in d1b552fabbe29aceb08307fab5dd55df9169b119